presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -31,4 +34,16 @@ protected QueryRunner createQueryRunner()
         mysqlServer = new TestingMySqlServer(false);
         return createMySqlQueryRunner(mysqlServer, CUSTOMER, NATION, ORDERS, REGION);
     }
+
+    @Test
+    public void testNativeLargeIn()
+    {
+        // 200_000 causes error:
+        // "com.mysql.jdbc.PacketTooBigException: Packet for query is too large (1488946 > 1048576).
+        //  You can change this value on the server by setting the max_allowed_packet' variable."
+        String longValues = range(0, 100_000)
+                .mapToObj(Integer::toString)
+                .collect(joining(", "));
+        mysqlServer.execute("SELECT count(*) FROM tpch.orders WHERE orderkey IN (" + longValues + ")");
```

Comment:
am i right this test fails (as per comment above)?

Reply:
for 100_000 it works. it fails with 200_000

Reply:
I'd rather see this as a predicate pushdown test that exercises Presto MySQL connector too.
(the connector can block large push downs, and then the test would become irrelevant to Presto codebase)

Reply:
> I'd rather see this as a predicate pushdown test that exercises Presto MySQL connector too.

It's very difficult to make tuple domain so large to be pushed to connector. Planning IN with 100_000 values takes forever in Presto.

> (the connector can block large push downs, and then the test would become irrelevant to Presto codebase)

I think it's still relevant as it's base for choosing what is the tuple domain simplification threshold (it can be part of commit message along with link to test case).